tests/db_upgrade2_test.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+"""auth_db db upgrade
+
+:copyright: Copyright (c) 2023 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+
+
+_UID_WITH_FLASH_ROLE = "pzHuDps6"
```

Comment:
document that this needs to be in sync with committed auth.db.

Reply:
8e7f059f1